dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/internal/WebAppSecurityConfigImpl.java

```diff
@@ -603,4 +630,19 @@ public long postParamMaxRequestBodySize() {
         return postParamMaxRequestBodySize.longValue();
     }
 
+    // This method is for config users that need to know if an admin has provided a true/false or no value for a
+    // boolean config attribute.  The current infrastructure does not properly support this
+    public static Boolean getBooleanValue(String attribute, String strValue) {
+      Boolean retVal = null;
+      if (strValue!=null && strValue.length()>0) {
+        if ("true".equalsIgnoreCase(strValue) || "false".equalsIgnoreCase(strValue)) {
+          retVal = Boolean.valueOf(strValue);
+        } else {
+          Object[] inserts = new Object[] { strValue, (attribute==null?"":attribute)}; 
+          Tr.warning(tc, "INVALID_BOOLEAN_ATTRIBUTE", inserts);
```

Comment:
Where does the `INVALID_BOOLEAN_ATTRIBUTE` NLS message exist?

Reply:
I removed that else section.